src/app/routes/utils/regex/utils/index.js

```diff
@@ -31,15 +33,28 @@ export const getArticleManifestRegex = services => {
 };
 
 export const getFrontPageRegex = services => {
-  const serviceRegex = getServiceRegex(services);
+  // if environment is not live then filter out and remove kyrgyz from list of services
+  let frontPageServices = services;
+  if (!isLive()) {
+    frontPageServices = services.filter(service => service !== 'kyrgyz');
+  }
+  const serviceRegex = getServiceRegex(frontPageServices);
   return `/:service(${serviceRegex}):variant(${variantRegex})?:amp(${ampRegex})?`;
 };
 
-export const getHomePageRegex = services => {
+export const getTipoHomeRegex = services => {
   const serviceRegex = getServiceRegex(services);
   return `/:service(${serviceRegex}):variant(${variantRegex})?/tipohome:amp(${ampRegex})?`;
 };
 
+export const getHomePageRegex = () => {
+  if (!isLive()) {
+    const homePageServiceRegex = getServiceRegex(['kyrgyz']);
+    return `/:service(${homePageServiceRegex}):variant(${variantRegex})?:amp(${ampRegex})?`;
+  }
+  return '';
```

Comment:
Let's try to remove this and see what happens? 🤔

Reply:
Removed